### DISCUSSION TITLE
How should a new Ubik look like?

### DISCUSSION THREAD

**Discussion by SniperGoth** [redeclipse/base#1162] [General]
I've been thinking that a clean sci-fi like setting would do good for it, since it already has very tall buildings to begin with, so it would fit like a glove, and ditch the industrial/dark look it currently has.

Maybe it should be expanded to have some more buildings to make it a bit more dense, with models, because right now that's also another issue of the map (the model scaling issue will need to be fixed for this to work tho, see https://github.com/red-eclipse/base/issues/845 )

Some reference images : 

![ec1a89b412f523cb981ce2ab80d9ba4d](https://user-images.githubusercontent.com/24359960/57698443-0a013280-762c-11e9-9345-2959f41c484f.png)
![trials_fusion___environment_concept_by_artofjokinen-d78qdu4-1](https://user-images.githubusercontent.com/24359960/57698444-0a99c900-762c-11e9-94ed-08158eab5404.jpg)
![27SWwide jpg85443215-02ef-41b0-bfdd-b5799fbd9f23Original](https://user-images.githubusercontent.com/24359960/57698445-0a99c900-762c-11e9-80fe-1726fb952a16.jpg)
![EVOOI8x](https://user-images.githubusercontent.com/24359960/57698928-26ea3580-762d-11e9-9da5-488486f8d845.jpg)

Any thoughts? Post your suggestions!

**Comment by no-lex** [comment]
As I did in #84 (Ennui), using flat textures with nonflat normals (e.g. from phillipk or trak) may allow you to get that really clean look while not being entirely flat and boring. 
![20190514073043](https://user-images.githubusercontent.com/23445929/57706108-f6e56700-7619-11e9-93b3-e299faee06e5.png)

Some normals to consider cribbing: 
```
data/phillipk/pk02/panels01a_n
data/phillipk/pk02/panels01b_n
data/phillipk/pk02/panels01c_n
data/phillipk/pk02/door02_n
data/phillipk/pk02/floor08b_n
data/phillipk/pk02/floor_cyl02_n
data/phillipk/pk02/wall02b_n

data/trak/trak5/trim1_n
data/trak/trak5/trim2_n
data/trak/trak5/trim4_n
data/trak/trak5/trim5_n
data/trak/trak5/trim7_n
data/trak/trak5/trim8_n
```
The base diffuse texture is unimportant. I used dziq's brushed metal, but phillipk's flat generic or nobiax's flat complex texture would likely work as well.

You can also change the specular maps if you wish, but the difference is fairly slight.
This is easiest to do by hand in the config file for the map.

**Comment by byronsharman** [comment]
For those interested, I found a YT video of the last concept art

https://youtu.be/hJ7jlGJ7JTg

**Comment by SniperGoth** [comment]
Just to add to nono's comment : 

[https://www.youtube.com/watch?v=8z1H5DRQ_L8](url)

So far, i arranged the layout a bit, so it's clearer where you are/people are. 

![20190514161244](https://user-images.githubusercontent.com/24359960/57935482-420cad80-7898-11e9-854a-6fe99eec3ed1.png)

And made these generic rails.
![20190515214703](https://user-images.githubusercontent.com/24359960/57935510-5650aa80-7898-11e9-854e-6dc321b1e990.png)
![20190515214614](https://user-images.githubusercontent.com/24359960/57935511-5650aa80-7898-11e9-9d08-ad83408410ab.png)

Still figuring out  how i'll populate the outside area, maybe with a mountain or two, and some buildings.

**Comment by lambda-11235** [comment]
Is the layout going to be different? My issue with the old Ublk was that on the lower level everything kinda looked the same and it wasn't immediately obvious how to go up.

edit: Looking at it I think my main problem was that there where only two central boosters to get up. That makes it so to go up one has to run to the center, and then run to wherever on the second level they want to go. It would be nice to have more routes.

**Comment by SniperGoth** [comment]
Yup, you can see in the first screenshot that the layout got a slight modification already on the left side.
Im considering adding more boosters, or even pathways to mix things up.
We will see down the line if fits or not.
